packages/ui/src/Alert.tsx

```diff
@@ -44,15 +44,25 @@ const isAlertActionButton = (action: AlertAction): action is AlertActionButton =
 
 export type AlertProps = {
   type: ToastType
-  title: string
-  content: ReactNode
+  title?: string | ReactNode
+  content?: string | ReactNode
```

Comment:
IIRC `ReactNode` already includes `string`:
```suggestion
  title?: ReactNode
  content?: ReactNode
```

Reply:
thanks